WMF Framework/RelatedSearchFetcher.swift

```diff
@@ -18,14 +18,6 @@ final class RelatedSearchFetcher: Fetcher {
             let thumbnail: Image?
             let coordinates: Coordinates?
 
-            var isList: Bool {
-                return description?.contains("Wikimedia list article") ?? false
-            }
-
-            var isDisambiguation: Bool {
-                return description?.contains("disambiguation page") ?? false
```

Comment:
Rather than further the use of these english-specific checks, it's probably better to remove them from `MWKSearchResult`.  They were only used to improve the quality of random unused elsewhere. I updated the random fetcher to filter on `isDisambiguation` and removed the `isList` sort. I'd argue that it's OK for random to show lists, but worth discussing.

Reply:
There's also this endpoint we could start using at some point -> https://en.wikipedia.org/api/rest_v1/#!/Page_content/get_page_random_format

Reply:
@joewalsh hmm... unsure why it's better to decrease the quality of the English results...

Reply:
@montehurd I like list articles. From my perspective, it's an increase in quality. Ultimately I'm fine with whatever's decided after consulting with the rest of the team - I've already implemented the switch to the upstream random endpoint that keeps similar heuristics: https://github.com/wikimedia/wikipedia-ios/tree/bug/random

Reply:
I'm fine if Carolyn and Josh want them back but they were not a good experience imo. Mind if we loop them in before changing this?

Reply:
Don't mind at all, that's what I was suggesting